packages/@n8n_io/eslint-config/package.json

```diff
@@ -1,21 +1,23 @@
 {
   "name": "@n8n_io/eslint-config",
   "private": true,
+  "version": "0.0.1",
   "devDependencies": {
     "@types/eslint": "8.4.6",
     "@typescript-eslint/eslint-plugin": "^5.36.2",
     "@typescript-eslint/parser": "^5.36.2",
-    "@vue/eslint-config-typescript": "^11.0.0",
+    "@vue/eslint-config-typescript": "^8.0.0",
     "eslint": "8.23.0",
     "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-diff": "^2.0.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-n8n-local-rules": "^1.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-vue": "^9.4.0"
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-vue": "^7.16.0"
```

Comment:
these versions were unused, as they were getting overridden in `design-system/package.json`. I copied over the same versions here, to avoid introducing any breaking dependencies.